### ISSUE TITLE
Review observations: activity

### ISSUE THREAD

**Issue by peterdesmet** [ices-tools-dev/esas#39]
`observations: activity` indicates the activity with an integer code:

https://github.com/ices-tools-dev/esas/blob/0d294b4ea678e34f52409a16602f9133ac179dcf/_data/table-schemas/observations.yaml#L577-L586

- [ ] @ices-tools-dev/data-and-information Does ICES already have a controlled vocabulary for this?
- [ ] @nicolasvanermen Wouldn't human readable values like `swimming` be better, or are the integer codes well established?

**Comment by HjalteParner** [comment]
@peterdesmet, yes ICES already have a platform code vocabulary including platform type or class if you like.

Please have a look at ICES platform code request system at http://vocab.ices.dk/request for details.

**Comment by peterdesmet** [comment]
Thanks @HjalteParner. I will indicate your remark in issue #16, as this one relates to "activity". 😊

**Comment by nicolasvanermen** [comment]
I would hold on to the integers, not familiar with this field myself, plus
it might be added with new values in the future, resulting in a much longer
list than the present one.

Nicolas Vanermen
*Wetenschappelijk medewerker*
Instituut voor Natuur- en Bosonderzoek
Havenlaan 88, bus 73, 1000 Brussel
0486/361.582

**Comment by Osanna123** [comment]
need new vocab

**Comment by Osanna123** [comment]
Question from RMG: 
Can Activity be integrated with Behaviour?

**Comment by nicolasvanermen** [comment]
Behaviour codes are mostly used during ship-based surveys, Activity codes
during aerial monitoring. But, some aerial surveyors will use both columns!
By integrating both columns, information will go lost.

Nicolas Vanermen
*Wetenschappelijk medewerker*
Instituut voor Natuur- en Bosonderzoek
Havenlaan 88, bus 73, 1000 Brussel
0486/361.582

**Comment by Osanna123** [comment]
@nicolasvanermen merging of the two reference lists does not necessarily mean change of the format. 
However, it does seem unnecessary having two fields, use of which is defined by the observation type. I'd then suggest 1 field, with an option to report multiple codes instead.

**Comment by nicolasvanermen** [comment]
Indeed the controlled vocabulary lists can in theory be merged. However, I
would definitely keep both fields in the data format. For almost 10.000
observations both fields are filled. Observation type mainly tries to
pinpoint the main focus of activity/behaviour scoring but does not exclude
the possibility to fill in both fields. Having said this, I think it is
also more logical to hold on to two controlled vocabulary lists.

Nicolas Vanermen
*Wetenschappelijk medewerker*
Instituut voor Natuur- en Bosonderzoek
Havenlaan 88, bus 73, 1000 Brussel
0486/361.582

**Comment by peterdesmet** [comment]
The latest status on the columns `behaviour` and `activity` is that they can be merged (already implemented in https://github.com/ices-tools-dev/esas/commit/231b09574447eac7b7a3998494cb70d184ca5e98), allowing multiple values if needed with `~`.

The column [behaviourRecording](https://ices-tools-dev.github.io/esas/tables/#behaviourrecording) can likely be removed, but I am waiting on confirmation from Ruben Fijn / @nicolasvanermen.
